src/main/java/org/openrewrite/java/testing/mockito/PowerMockitoMockStaticToMockito.java

```diff
@@ -446,7 +446,8 @@ private J.ClassDeclaration addFieldDeclarationForMockedTypes(J.ClassDeclaration
                                 new Cursor(getCursor().getParentOrThrow(), classDecl),
                                 classDecl.getBody().getCoordinates().firstStatement(),
                                 classlessTypeName,
-                                classlessTypeName
+                                // Not sure if replacing `.` with `_` is a structural solution, but it works for inner classes for now
```

Comment:
This seems more like something to post as a self-review comment as opposed to a comment to carry forward in the code.
```suggestion
```